src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -36,14 +36,15 @@ class Router extends BaseRouter implements WarmableInterface, ServiceSubscriberI
     private $paramFetcher;
 
     /**
-     * @param ContainerInterface      $container  A ContainerInterface instance
-     * @param mixed                   $resource   The main resource to load
-     * @param array                   $options    An array of options
-     * @param RequestContext          $context    The context
-     * @param ContainerInterface|null $parameters A ContainerInterface instance allowing to fetch parameters
+     * @param ContainerInterface      $container     A ContainerInterface instance
+     * @param mixed                   $resource      The main resource to load
+     * @param array                   $options       An array of options
+     * @param RequestContext          $context       The context
+     * @param ContainerInterface|null $parameters    A ContainerInterface instance allowing to fetch parameters
      * @param LoggerInterface|null    $logger
+     * @param string|null             $defaultLocale The default locale
```

Comment:
We prefer not adding annotations when the signature is explicit enough. Looks like that the case in this PR. That's good news as removing these lines will withdraw the need to update all other lines :)

Reply:
Done :)